lldb/source/Plugins/ObjectFile/Mach-O/ObjectFileMachO.cpp

```diff
@@ -2152,80 +2152,6 @@ UUID ObjectFileMachO::GetSharedCacheUUID(FileSpec dyld_shared_cache,
   return dsc_uuid;
 }
 
-static SymbolType GetSymbolType(
```

Comment:
Same comment as in the ABI file above — this should most likely go into llvm.org instead.

Reply:
this overload of `GetSymbolType` does not exist in mainline llvm.